device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/Mqtt.java

```diff
@@ -439,11 +413,6 @@ public void deliveryComplete(IMqttDeliveryToken iMqttDeliveryToken)
         }
     }
 
-    public Pair<String, byte[]> peekMessage()
-    {
-        return this.allReceivedMessages.peek();
-    }
```

Comment:
No need for a function this simple, so the places that called this previously now just call this.allReceivedMessages.peek();